File: backend/models/invoice.rb
class Invoice
  attr_reader :billing_address, :shipping_address, :order

  def initialize(order:, billing_address:, shipping_address:)
    @order = order
    @billing_address = billing_address
    @shipping_address = shipping_address
  end

  def shipping_label
    "#{order.customer.name}\nCEP: #{shipping_address.zipcode}\nItem isento de impostos conforme disposto na Constituição Art. 150, VI, d."
  end
end

File: backend/Solucao Creditas Challenge/PythonApplication1/invoice.py
from order import OrderTotal
from customer import Customer

class Invoice:
    billing_address = None
    shipping_address = None
    order = None
    total = None

    def __init__(self, Customer, OrderTotal, amountFinal):
    
        self.billing_address = Customer.zipCodeBilling
        self.shipping_address = Customer.zipCodeShipping
        self.order = OrderTotal.items
        self.total = amountFinal